ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -1490,3 +1490,66 @@ def test_tensorflow_avg_pool1d(
         strides=strides,
         padding=padding,
     )
+
+
+@st.composite
+def _pool_args(draw):
+    dims = draw(st.integers(min_value=1, max_value=3))
+    data_formats = ["NWC", "NHWC", "NDHWC"]
+    data_format = data_formats[dims - 1]
+    pooling_types = ["AVG", "MAX"]
+    pooling_type = pooling_types[draw(st.integers(min_value=0, max_value=1))]
+    return (
+        draw(
+            helpers.arrays_for_pooling(
+                min_dims=dims + 2,
+                max_dims=dims + 2,
+                min_side=1,
+                max_side=4,
+            )
+        ),
+        pooling_type,
+        data_format,
+        dims + 2,
+    )
+
+
+# pool
+@handle_frontend_test(
+    fn_tree="tensorflow.nn.pool",
+    x_k_s_p_df=_pool_args(),
```

Comment:
Can you test whether this works with the helper `get_arrays_for_pooling(...)` as used previously in [here](https://github.com/unifyai/ivy/pull/15884/files#diff-259a7991f1af0d34dea0190b1b676d36560034f3d2075c8337f6d6c7c08225e2R1468-R1470)

Reply:
Hi, @ZiadAmerr, it seems like this function ```get_arrays_for_pooling(...)``` doesn't exist in my current version of the repo.

Reply:
Sorry, it is `helpers.arrays_for_pooling(...)`.

You can find it in the link of my previous comment